.github/renovate-config.json5

```diff
@@ -3,10 +3,11 @@
   onboarding: false,
   requireConfig: "ignored",
   platform: "github",
-  repositories: ["jenkinsci/helm-charts"],
+//  repositories: ["jenkinsci/helm-charts"],
```

Comment:
```suggestion
  repositories: ["jenkinsci/helm-charts"],
```

Reply:
Whoops, yeah, thanks for catching this. Had to comment that to run Renovate locally.